oraclebmc/util.py

```diff
@@ -99,3 +100,14 @@ def __bool__(self):
         return self.truthy
     # PY2 Compatibility
     __nonzero__ = __bool__
+
+def initkwargs(fn):
+    @functools.wraps(fn)
+    def init(self, **kwargs):
+        fn(self)
+        for kw, value in kwargs.items():
+            if kw not in self.swagger_types:
```

Comment:
This is a small nitpick, but swapping the iteration to loop over `swagger_types` instead of `kwargs.items()` is slightly faster and provides a more thorough error message.

Right now this will only display the first unknown attribute, instead of all the unknown attributes.  By iterating the `swagger_types` and popping known keys, `kwargs` will only be non-empty if there are unknown attributes.  This could eventually include required/optional logic and fail fast on missing arguments.

Using a `Sentinel` to skip available but not provided swagger types:

```python
missing = Sentinel("Missing")

for attr in self.swagger_types:
    value = kwargs.pop(attr, missing)
    if value is not missing:
        setattr(self, attr, value)
if kwargs:
    raise(...)
```

For 2.7/3+ compatibility, you'll probably want `six.iterkeys` in the `raise`.